tools/WixBuild.props

```diff
@@ -228,7 +228,12 @@
     On the build machine the OFFICIAL_WIX_BUILD env var will point to the private key, otherwise
     use the public key in the enlistment.  We must find a key file.
   -->
-  <PropertyGroup Condition=" '$(OFFICIAL_WIX_BUILD)'=='' ">
+  <PropertyGroup Condition=" '$(DisableSigning)' == 'true' ">
```

Comment:
These property groups are not `if... else` statements, so this gets overwritten for `OFFICIAL_WIX_BUILD` below and breaks the build.